runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTabPanel.java

```diff
@@ -209,6 +240,30 @@ public void update()
 					farmingContractIcon.setToolTipText("");
 				}
 			}
+
+			String configKey = NOTIFY_CONFIG_KEY +
+				"." + patch.getRegion().getRegionID() +
+				"." + patch.getVarbit().getId();
+
+			JButton toggleNotify = panel.getNotifyIcon();
+
+
+			ImageIcon notifyIcon = new ImageIcon(ImageUtil.resizeImage(itemManager.getImage(ItemID.BELL_BAUBLE_6847), 18, 16));
```

Comment:
probably should use a greyscale icon from fontawesome or somewhere.
https://fontawesome.com/icons/bell?style=solid and https://fontawesome.com/icons/bell-slash?style=regular